src/tools/rest-frontend/README.md

```diff
@@ -6,15 +6,15 @@ This document aims to provide information about Elektras `rest-frontend`, which
 
 ## Design and Structure
 
-The frontend is developed as Single Page Application in [AngularJS (v1.5)](https://angularjs.org/). All dependencies are either already contained in the application project or (preferred) resolved through [npm](https://www.npmjs.com/) and [bower](https://bower.io/) during installation (requires active internet connection). Compiling (concatenation & minification), as well as other tasks like running a lightweight webserver are handled by the task runner `grunt`.
+The frontend is developed as Single Page Application in [AngularJS (v1.5)](https://angularjs.org/). All dependencies are either already contained in the application project or (preferred) resolved through [npm](https://www.npmjs.com/) during installation (requires active internet connection). Compiling (concatenation & minification), as well as other tasks like running a lightweight webserver are handled by the task runner `grunt`.
```

Comment:
It should read "single-page application".

At how many places is a dep to AngularJS 1.5 and what is the effort to migrate to AngularJS 2?